extensions/blocks/story/story.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Story Block.
+ *
+ * @since 8.6.1
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Story;
+
+use Jetpack_AMP_Support;
+use Jetpack_Gutenberg;
+
+const FEATURE_NAME = 'story';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers the block for use in Gutenberg
+ * This is done via an action so that we can disable
+ * registration if we need to.
+ */
+function register_block() {
+	jetpack_register_block(
+		BLOCK_NAME,
+		array( 'render_callback' => __NAMESPACE__ . '\render' )
+	);
+}
+add_action( 'init', __NAMESPACE__ . '\register_block' );
+
+/**
+ * Render story block
+ *
+ * @param array $attributes  Block attributes.
+ *
+ * @return string
+ */
+function render( $attributes ) {
+	Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
+
+	$get_image_template = function( $media ) {
+		if ( ! isset( $media['id'] ) || ! isset( $media['url'] ) ) {
```

Comment:
I think we could use `empty()` here, to cover more use-cases. Same below.

```suggestion
		if ( empty( $media['id'] ) || empty( $media['url'] ) ) {
```